app/services/firebase/push_notification_service.rb

```
@@ -0,0 +1,213 @@
+      update_notification_status
+    end
+    
+    {
+      success: success_count > 0,
+      message: "Đã gửi thành công #{success_count}/#{device_tokens.count} notifications",
+      total_tokens: device_tokens.count,
+      success_count: success_count,
+      failed_count: device_tokens.count - success_count,
+      results: results
+    }
+  end
```

RuboCop (1.81.6)

[convention] Assignment Branch Condition size for `send_to_multiple_devices` is too high. [<8, 26, 13> 30.15/23]

(Metrics/AbcSize)